pkg/operator/controllers/subnets/subnets_controller.go

```diff
@@ -30,8 +30,9 @@ import (
 )
 
 const (
-	CONFIG_NAMESPACE string = "aro.azuresubnets"
-	ENABLED          string = CONFIG_NAMESPACE + ".enabled"
+	CONFIG_NAMESPACE         string = "aro.azuresubnets"
+	NSG_ENABLED              string = CONFIG_NAMESPACE + "nsg.enabled"
```

Comment:
I believe that we should have:

`aro.azuresubnets.enabled`
`aro.azuresubnets.nsg.managed`
`aro.azuresubnets.serviceendpoint.managed`

If we have the controller enabled but nothing under it managed, it may still emit statuses (e.g. that the present state is invalid) without doing anything to change it.

Reply:
Updated.